src/router/bundle/bundleParser.test.ts

```diff
@@ -790,7 +790,8 @@ describe('parseResource', () => {
                                 reference: `${serverUrl}/Patient/111`,
                             },
                             device: {
-                                reference: `${serverUrl}/Device/222`,
+                                // make sure versioned references also work
+                                reference: `${serverUrl}/Device/222/_history/7`,

```

Comment:
Just to understand this testcase. This is POSTing a bundle with a historical reference. The output of calling this method should have that reference with `_history/7`, right?

Maybe I am not understanding this testcase

Reply:
I suppose it should. 
The main goal of this change was to avoid calling vReadResource with a vid that's not formatted correctly. 
But obviously we also want to store the reference with the version info. Maybe I don't understand the question correctly.